easybuild/easyblocks/g/gromacs.py

```diff
@@ -182,6 +182,12 @@ def sanity_check_step(self):
         if self.toolchain.options.get('usempi', None):
             suff = '_mpi'
 
+        # Add the _d suffix to the suffix, in case of the double precission
+        if '-DGMX_DOUBLE=on' in self.cfg['configopts']:
+            suff = suff + '_d'
+        elif '-DGMX_DOUBLE=ON' in self.cfg['configopts']:
+            suff = suff + '_d'
```

Comment:
I tackled this in #960 by adding a custom easyconfig parameter `double_precision`, and using a regex to check whether `GMX_DOUBLE` is set, cfr. 812c304